packages/core/admin/admin/src/content-manager/components/RepeatableComponent/DraggedItem/index.js

```diff
@@ -158,6 +159,16 @@ const DraggedItem = ({
     }
   }, [isDragging, setIsDraggingSibling]);
 
+  useEffect(() => {
```

Comment:
Why is this necessary? I would include comments why you're doing it :)

Reply:
@joshuaellis Thanks for reviewing. We updated PR to focus on the first focusable element of new added item only. IMO it provides a good UX although using `querySelector` is not a very good idea but we couldn't find a good way to get the `ref` for first field. Do you any suggestions for this? 

cc @ronronscelestes

Reply:
It's a nice idea but I don't think we should be doing that, we typically use W3.org as basis for accessible patterns and they don't do this in their example – https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html

I haven't done extensive research into this, so if you do have an ARIA orientated document example you can share where this is advised, i'd be happy to read it.

Reply:
@joshuaellis I can see that our Accordion is already implemented with common pattern but we are focusing on "new item" only in this issue. BTW if the way of using `querySelector` is not good enough, another way is to keep focus state on the Accordion title of new item but it's empty div, so it gives this weird UI

<img width="667" alt="image" src="https://user-images.githubusercontent.com/1501599/207897455-15db641c-0166-46f8-b3de-d86952d494d7.png">

WDYT if we show something like placeholder text (like three dots or `&nbsp;`), we would have a better UI like

<img width="769" alt="image" src="https://user-images.githubusercontent.com/1501599/207898410-cb2a1f24-0631-4771-8acd-0effec51282f.png">

Reply:
I think the focus should probably be on the caret button, but this change would have to be in the DesignSystem repo instead of here. You can open an issue there to discuss it further, for now i would leave it. Putting in a random label for the button probably isn't best for screen readers... But you have highlighted an accessibility issue we need to address.

Reply:
Thanks @joshuaellis  
The changes relating to moving focus to newly added entry has been reverted in this PR, to be further discussed on the design system repo.
Change left is relating to making the `Add an entry` button focusable using the keyboard. kindly take a look at it.